frontend/src/app/AboutDialog.scss

```diff
@@ -3,7 +3,7 @@
 // Override to use the full width of the dialog rather than wrapping early
 .odh-about-dialog {
```

Comment:
[patternfly/patternfly/issues/6871](https://github.com/patternfly/patternfly/issues/6871) has been resolved in v6. Once the fix has been verified on this branch, `AboutDialog.scss` and any related references can be removed as part of [RHOAIENG-9742](https://issues.redhat.com/browse/RHOAIENG-9742).

Reply:
Opened a PR on your branch @jeff-phillips-18 https://github.com/jeff-phillips-18/odh-dashboard/pull/40